load/UpdateFileset.py

```diff
@@ -0,0 +1,84 @@
+# UpdateFileset.py
+
+from Config import *
+from LPTSExtractReader import *
+from SQLUtility import *
+from SQLBatchExec import *
+
+class UpdateFileset:
+    def __init__(self, config, db, dbOut, lptsReader):
+        self.config = config
+        self.db = db
+        self.dbOut = dbOut
+        self.lptsReader = lptsReader
```

Comment:
I have modified it because to achieve to test it locally using the command: `python3 load/UpdateFileset.py test`. But, I was debugging the project and I think we are not using this file into ETL flow. You can check it because you can not currently see any invocation to `UpdateFileset` class.

Reply:
This is actually Stage D, so shouldn't even be in the codebase yet... please delete until we get closer to Stage D

Reply:
This is actually Stage D, so shouldn't even be in the codebase yet... please delete until we get closer to Stage D

Reply:
I understand @bradflood, It has been deleted.